src/Models/UserModel.ts

```diff
@@ -0,0 +1,63 @@
+import { Schema, Document, model, Model } from "mongoose";
+
+export interface IUser {
+  firstname: string;
+  lastName: string;
+  fullName: string;
+  phoneNumbers?: string;
+  age?: number;
+  gender?: string;
+  email: string;
+  userDeleted?: boolean;
+  clerkId: string;
+}
+
+export interface IUserDocument extends IUser, Document {
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+const UserSchema = new Schema<IUserDocument>(
+  {
+    clerkId: {
+      type: "string",
```

Comment:
Is there something wrong with this type?

Reply:
i am not sure, i kept in order to be out of any problem if we need it for some referencing in future.